lib/src/base-styles/_nx-alert.scss

```diff
@@ -61,10 +61,10 @@
 
 .nx-alert--error {
   background-color: #fff6f8;
-  border-color: #cd0028;
+  border-color: $nx-red-800;
```

Comment:
Should we make `$nx-error` common variables for here and the buttons (and any other error states)?

Reply:
> We should make $nx-error common variables for here and other error states.

I definitely think we should go through and improve and cleanup the more semantic color variables, but not in this ticket.